target/linux/rockchip/image/armv8.mk

```diff
@@ -23,3 +23,12 @@ define Device/pine64_rockpro64
   IMAGE/sysupgrade.img.gz := boot-common | boot-script | pine64-img | gzip | append-metadata
 endef
 TARGET_DEVICES += pine64_rockpro64
+
+define Device/radxa_rock-pi-4
+  DEVICE_VENDOR := Radxa
+  DEVICE_MODEL := Rock Pi 4
+  SOC := rk3399
+  UBOOT_DEVICE_NAME := rock-pi-4-rk3399
+  IMAGE/sysupgrade.img.gz := boot-common | boot-script | pine64-img | gzip | append-metadata
```

Comment:
Have you tried adding `brcmfmac-firmware-4329-sdio` and `kmod-brcmfmac` as packages to enable the WiFI?

Reply:
brcmfmac-firmware-4329-sdio does't work, I have the WiFi working with brcmfmac-firmware-43456-sdio from armbian, can I add this to the PR https://github.com/mj22226/openwrt/commit/95564b855f3daa8ac5769c4bc7028d8fc1537999